Documentation/build-container-examples/postgres/build-postgres.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+set -e
+
+if [ "$EUID" -ne 0 ]; then
+    echo "This script uses functionality which requires root privileges"
+    exit 1
+fi
+
+PGDATA=/var/lib/postgresql/data
+
+# Start the build with an empty ACI
+acbuild --debug begin
+
+# In the event of the script exiting, end the build
+trap "{ export EXT=$?; acbuild --debug end && exit $EXT; }" EXIT
+
+# Name the ACI
+acbuild --debug set-name example.com/postgres
+
+# Based on alpine
+acbuild --debug dep add quay.io/coreos/alpine-sh
+
+# Install postgres and bash
+acbuild --debug run -- apk update
+acbuild --debug run -- apk add postgresql bash
+
+# Create postgres data directory
+acbuild --debug run -- mkdir -p $PGDATA
+acbuild --debug run -- chown -R postgres:postgres $PGDATA
+
+# Create postgres run directory
+acbuild --debug run -- mkdir -p /var/run/postgresql
+acbuild --debug run -- chown -R postgres:postgres /var/run/postgresql
+acbuild --debug run -- chmod 2777 /var/run/postgresql
```

Comment:
why 2777? shouldn't belonging to postgress be good enough?

Reply:
right, I'll remove it